site/src/load.rs

```diff
@@ -160,9 +188,14 @@ impl SiteCtxt {
             }
         };
 
+        let master_commits = MasterCommitCache::download()
+            .await
+            .unwrap_or(MasterCommitCache::empty()); // still run the site if we can't get the commits right now
```

Comment:
This should probably be a regular unwrap() -- I'd rather not change the behavior we had before in terms of the *initial* fetch of commits. In other words, stale commits are probably fine (bounded by ~60 seconds), but we should not have just non-existent data.

Reply:
Well, before caching, the failure mode was an error when accessing the compare page, the site wouldn't outright die on startup, but if that is preferred I can make the change.